public/data/2042400-2724/info.json

```diff
@@ -0,0 +1,50 @@
+{
+    "user": {
+      "username": "Alexandra Lerma",
+      "profileImage": "https://lh3.googleusercontent.com/a/ACg8ocLQw2Ztgi6CaRswuIyRoyrb_LzCqfYM24mnpZXRkAgJJsNgz44=s288-c-no",
+      "socialLink": "https://www.instagram.com/moonloveless/"
```

Comment:
only @MoonLoveless not the complete URL please :)

Reply:
Dont Worry teacher, Alexandra fix it the problem